src/uniprotkb/types/uniSave.ts

```diff
@@ -0,0 +1,43 @@
+export type UniSaveVersion = {
+  accession: string;
+  database: string; // 'Swiss-Prot' | 'TrEMBL';
```

Comment:
```suggestion
  database: 'Swiss-Prot' | 'TrEMBL';
```
Any reason to not narrow it down now?

Reply:
Nope, at the beginning I was just worried some old data might be different, but looks like it's always those 2, so I'll take your suggested change